### ISSUE TITLE
Crash to desktop when minimizing in Fullscreen mode

### ISSUE THREAD

**Issue by Parahacker** [CleverRaven/Cataclysm-DDA#23862]
Game version: experimental 7426

Operating system: Win 10

Tiles or curses: Tiles

Mods active: All of them that are bundled; except Dino mod, generic guns, and the balancing mods except for the default (filthy clothes etc.)

### Expected behavior

Program minimizes without issue

### Actual behavior

crash to desktop
### Steps to reproduce the behavior

Set fullscreen mode, then minimize or switch applications; happens every time without fail, any number of mods, any other settings in graphics are irrelevant. It happens when fullscreen, does not happen when not fullscreen (borderless windowed does not crash).

On a related note, I use fullscreen for one reason: I can't 'zoom in' the UI, only the map. On my screen that means I can't read the text. I could change my entire display resolution, but that's a pain and I have to reset it to normal again when not playing CDDA. I'm creating another topic for a zoom feature request.

**Comment by Parahacker** [comment]
More detail: I don't know if this is helpful or not, but when I minimize / switch apps, the screen will go to that app/desktop for a few seconds, then flash black for a few seconds, like blinking - it happens 3 times in a row. It does this every time.

**Comment by BevapDin** [comment]
What does the debug log ("config/debug.log") contain?

**Comment by Parahacker** [comment]
Ok, Windows 10 seems to be actively hostile to me reading things. Now, when I change the screen resolution for my monitor through windows, instead of 'fitting' the screen it just decreases the size of the visible portion.

Also, now I can't get CDDA to 'zoom' by going fullscreen no matter what settings I use in CDDA or in compatability settings. I've tried everything.

I'd like to close this ticket and focus on the zoom-for-text ticket, because I can't play this game without being able to read it. The crash is immaterial at this point, the setting (fullscreen) that causes it now provides no benefit at all over borderless windowed which *doesn't* crash.

**Comment by Night-Pryanik** [comment]
@Parahacker is this still an issue after #23995?

**Comment by MisterFox** [comment]
Experiencing this issue in different circumstances.

Game version: Experimental 7502
Operating system: Windows 10 (version 1709)
Tiles or curses: Curses

I don't use fullscreen mode, but whenever I hit minimize, I'm stricken with a runtime error in Visual C++. There aren't any useful details in the runtime report (just says 'runtime error', directory of cataclysm.exe, the program requested termination in an unusual way)

**Comment by kevingranade** [comment]
No confirmation that the issue remains in a reproduceable way, please open a new issue if it does occur with the current version.
